internal/database/sqlite_migrate.go

```diff
@@ -929,6 +930,9 @@ CREATE INDEX indexer_identifier_index
 	`
 	ALTER TABLE action
 		ADD COLUMN skip_hash_check BOOLEAN DEFAULT FALSE;
+    
+    ALTER TABLE action
+        ADD COLUMN first_last_piece_prio BOOLEAN DEFAULT false;
```

Comment:
When adding new columns we only have to add them last as a new migration.

Reply:
Oops. Fixed!